run/kpp_standalone_interface.yml

```diff
@@ -0,0 +1,78 @@
+@@ -0,0 +1,81 @@
+active_cells:
+   - LosAngeles
+   - McMurdo
+   - Paris
+   - Beijing
+   - Kinshasa
+   - Kennaook
+   - Graciosa
+   - Utqiagvik
+   - Ozarks
+   - Amazon
+   - Congo
+   - Borneo
+   - IndianOcean
+   - AtlanticOcean
+   - PacificOcean
+   - ElDjouf
+settings:
+  output_dir: "./"
```

Comment:
It would be better to make the default output dir a subdirectory. When I ran this out-of-the-box my run directory got very cluttered.

Reply:
That sounds good, I can make the default output directory ./OutputDir.  Would it be better to create this directory if it doesn't exist?

Reply:
For GCHP and GC-Classic this directory will always exist in the run directory.

Reply:
Thanks for the clarification! For my GEOS runs, this was writing to a subdirectory named scratch in the run folder. I updated this to be ./OutputDir/ by default and added a comment in the YAML file.